packages/app-desktop/gui/Sidebar/styles/sidebar-sync-button.scss

```
@@ -13,4 +32,29 @@
 			animation: none;
 		}
 	}
+
+	&:not(.-syncing).-synced > .icon {
+		animation: icon-fade-in-a 300ms ease-in-out;
+		font-size: 0.85em;
+	}
+
+	&:not(.-syncing):not(.-synced) > .icon {
```

Stylelint (17.7.0)

[error] Expected complex :not() pseudo-class notation (selector-not-notation)

(selector-not-notation)